src/pkg/utils/utils.go

```diff
@@ -288,7 +292,7 @@ func GetContainerNamePrefixForImage(image string) (string, error) {
 
 func GetDefaultImageForDistro(distro, release string) string {
 	if _, supportedDistro := supportedDistros[distro]; !supportedDistro {
-		distro = "fedora"
+		distro = viper.GetString("general.default_distro")
```

Comment:
Umm... is this correct? The `fedora` here is meant as a hard-coded *fallback* that will always work. That's why it also can't be changed through the command line, because otherwise we can't guarantee it. Hence it shouldn't be influenced by the configuration file either.

Reply:
I'm unsure about this exact behaviour. Somebody may request some unsupported distribution but what they get is Fedora. In such a case I'd expect the tool to yell at me that the requested distro is not supported instead falling back to Fedora.

Reply:
Hmm... maybe I went a bit overboard here. Still, I'm not entirely sure we want to have a hard-coded fallback to Fedora. Yes, Fedora is our default but the user may want something else. But I may be wrong here.

If we keep the current behaviour, we could drop the following check with the panic because we can be certain the fallback will save the day.

Reply:
Whether we make the hard-coded fallback configurable or not is a different discussion. Currently it's not tweakable through the command line interface, so there's no need to expose it through the configuration file.

The way things stand at the moment, `fedora` is the only feasible hard fallback. It's the only fully supported community driven image. So if somebody runs `toolbox create` on Arch or Debian, we really don't have any other option than to fall back to `fedora`. :)

Once supported images for Arch or Debian are in place, the code won't need to hit the fallback.